src/components/Calculator.js

```diff
@@ -1,55 +1,47 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
+const Calculator = () => {
```

Comment:
- [ OPTIONAL] Kindly always use a  camelCase notation for the function component instead of Pascal case notation you can visit [js naming convention](https://www.w3schools.com/js/js_conventions.asp)